fmf/plugins/pytest/constants.py

```diff
@@ -0,0 +1,13 @@
+CONFIG_POSTPROCESSING_TEST = "test_postprocessing"
+PYTEST_DEFAULT_CONF = {
+    CONFIG_POSTPROCESSING_TEST: {
+        "test": """
+cls_str = ("::" + str(cls.name)) if cls.name else ""
+escaped = shlex.quote(filename + cls_str + "::" + test.name)
+f"python3 -m pytest -m '' -v {escaped}" """
```

Comment:
I'm not sure about pytest as a require :/ As it is now it is a 'hidden' require - you need to remember. But on the other hand it leaves up to user whether rpm or pip install is better.

Reply:
yes, it is now hidden, you are rigt, and theoretically it is also why I've let it configurable, not hardcoded.  in my project `fmf_metadata` I have own configs for upstream and dowstream data generation.  and in case you have just unittests then you an execute it directly via python `unittest` executor, or via some `nose` so that pytest is just default way.